R/hello.R

```diff
@@ -26,10 +26,12 @@ hipercow_hello <- function(progress = NULL, timeout = NULL, root = NULL) {
   ## TODO: here we also want to modify the template to use the
   ## fast-but-short queue; we can't do that until mrc-4801 is done
   ## though.
-  id <- task_create_expr(utils::sessionInfo(), submit = TRUE, root = root)
+  id <- task_create_expr(hipercow:::hipercow_handshake(),
+                         submit = TRUE, root = root)
   ok <- task_log_watch(id, timeout = timeout, progress = progress, root = root)
   result <- task_result(id, root = root)
   if (ok) {
+    hipercow_speak(system.file("comms/moo2.wav", package = "hipercow"))
```

Comment:
Use the hipercow_file helper from the other suggestion.

Also, what if we randomly selected the moo?

Reply:
Will save random mooing for later when we have more moos